acceptance/status/status_test.go

```diff
@@ -0,0 +1,48 @@
+package status_test
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/puppetlabs/pdkgo/acceptance/testutils"
+	"github.com/stretchr/testify/assert"
+)
+
+const APP = "prm"
+
+func Test_PrmStatus_NoArgs(t *testing.T) {
+	testutils.SkipAcceptanceTest(t)
+	if runtime.GOOS == "darwin" {
+		t.Skip("Docker based acceptance tests currently fail on MacOS")
+	}
```

Comment:
Skipping for now, turning on acceptance tests that require docker on MacOS should possibly be a separate ticket. Thoughts, @puppetlabs/devx?

Reply:
I think we skip this test, based on what you said the stand up time for the image would be to enable Docker.